data/src/main/java/com/teamagam/gimelgimel/data/alerts/repository/AlertsDataRepository.java

```diff
@@ -16,7 +16,7 @@
 
     @Inject
     public AlertsDataRepository() {
-        mReplayRepository = ReplayRepository.createReplayAll();
+        mReplayRepository = ReplayRepository.createReplayCount(1);
```

Comment:
I understand why you changed this, but what if someone needs to get all of the alerts?